src/components/input/demoTextarea/index.html

```diff
@@ -0,0 +1,22 @@
+<div layout="column" ng-cloak class="md-inline-form">
+
+  <md-content layout-padding>
```

Comment:
layout-padding is making the text areas look weird, wrap them with divs or make a class